app/js/components/rule/ruleGroupCard/ruleGroupCard.jade

```diff
@@ -1,17 +1,17 @@
 .card-group(ng-keypress='keyPressed($event)')
   header.header
-    .count(translate, translate-n='plugin.hitCount', translate-plural="{{$count}} Systems") 1 System
+    .count(translate, translate-n='plugin.hitCount', translate-plural="{{:: $count}} Systems") 1 System
     .row
       .col-sm-9
         i.fa.fa-users(aria-hidden="true")
         strong
           span.text-uppercase(translate) Group: &nbsp;
           span(ng-if='plugin.categories.length === 1')
             i.fa.category-icon.inline-right(ng-class="plugin.categories[0].toLowerCase()")
-            | {{plugin.categories[0]}} 
-          span &nbsp; {{plugin.display_name}}
+            | {{:: plugin.categories[0]}}
```

Comment:
These cannot be static because then switching to a different rule in a card view has no effect

Reply:
Wait... some of that file a was messed up. But specifically {{:: plugin.categories[0]}} can be otb.
I mean that code only checks the zero-th element anyway so it would always fail if otb didnt work.

I think line 11 is okay, can you ack?

Reply:
correct